Invoke-Tests.ps1

```diff
@@ -1,4 +1,4 @@
-﻿#Requires -Module @{ ModuleName = 'pester'; ModuleVersion = '5.3.1' }
+#Requires -Module @{ ModuleName = 'pester'; ModuleVersion = '5.3.1' }

```

Comment:
Don't change the encoding on the tests.
The PowerShell scripts should always be utf-8 with BOM encoded, so they are compatible on Windows PowerShell even if we use non-ascii letters.

Reply:
We got this fixed up during our review.